inetum-realjobs-frontend/src/app/services/application.service.ts

```diff
@@ -0,0 +1,19 @@
+import {Injectable} from "@angular/core";
+import {HttpClient} from "@angular/common/http";
+import {Observable} from "rxjs";
+import {Application} from "../models/application";
+import {ApplicationStatus} from "../models/application-status.enum";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ApplicationService {
+
+  constructor(private httpClient: HttpClient) {
+  }
+
+  updateApplication(id: number, status: ApplicationStatus): Observable<Application> {
+    return this.httpClient.put<Application>(`http://localhost:8080/api/applications/${id}/status`, {id, status});
```

Comment:
Same as controller, but why mention id twice?

Reply:
That's how REST specifies it and it became an habit, no other reason.